go.mod

```diff
@@ -23,8 +23,9 @@ replace (
 	github.com/crewjam/saml => github.com/crewjam/saml v0.4.1
 	github.com/docker/distribution => github.com/docker/distribution v2.7.1+incompatible
 	github.com/docker/docker => github.com/docker/docker v20.10.6+incompatible
-	github.com/rancher/rancher/pkg/apis => github.com/rancher/rancher/pkg/apis v0.0.0-20211218171736-3c1d5fac3248
-	github.com/rancher/rancher/pkg/client => github.com/rancher/rancher/pkg/client v0.0.0-20211218171736-3c1d5fac3248
+	github.com/rancher/rancher => /Users/annablendermann/RANCHER_REPOS/src/rancher
+	github.com/rancher/rancher/pkg/apis => /Users/annablendermann/RANCHER_REPOS/src/rancher/pkg/apis
+	github.com/rancher/rancher/pkg/client => /Users/annablendermann/RANCHER_REPOS/src/rancher/pkg/client
```

Comment:
The operators need to be updated in Rancher first. Then, in Rancher, run `go generate`.

After that has been merged in Rancher, then you can update the commits here.

Reply:
Done! Waiting on a review for https://github.com/rancher/rancher/pull/36461

Reply:
Merged. Updating the commits now...

Reply:
I updated the terraform provider commits with the top commit from rancher/rancher:release/v2.6, which is my merge commit for https://github.com/rancher/rancher/pull/36461.

Let me know if that's correct!